.github/workflows/deploy.yml

```diff
@@ -1,6 +1,9 @@
 # documentation: https://help.github.com/en/articles/workflow-syntax-for-github-actions
 name: deploy documentation (only on push to main branch)
 on:
+  schedule: 
```

Comment:
Actually, I don't think we want this in the regular `deploy.yml`. Having a cronjob that directly deploys the API docs means we have no human in the loop to check the changes to the API docs, and whether they make sense. I think the way this was implemented for the software overview in https://github.com/EESSI/docs/blob/main/.github/workflows/update_available_software.yml makes more sense for now: do a build of the docs, then create a PR based on that. It means the auto-generated docs will actually be _part_ of this repo, i.e. the physical *.md files will be files within the `EESSI/docs` repo. That'll allow us to do a check on them before they get deployed.

Long term, we might want to do this fully automatically, without human in the loop. But to start with, I think the human in the loop would be a good thing (even if someone needs to keep an eye on those PRs and merge them).